libs/filamat/src/MaterialBuilder.cpp

```diff
@@ -894,7 +894,7 @@ bool MaterialBuilder::generateShaders(JobSystem& jobSystem, const std::vector<Va
                         .domain = mMaterialDomain,
                         .materialInfo = &info,
                         .hasFramebufferFetch = mEnableFramebufferFetch,
-                        .usesClipDistance = v.variant.hasInstancedStereo(),
+                        .usesClipDistance = v.variant.hasStereo(),
```

Comment:
how will you distinguish between the two here, since multiview doesn't need clipDistance.

Reply:
I'm going to add a new param to material later, so the MaterialInfo structure above will include a type of stereoscopic as a member variable.